netbox/project-static/styles/overrides/_tabler.scss

```diff
@@ -52,7 +52,7 @@ pre {
 }
 
 // Altering background colors
-.page, .page-tabs .nav-tabs .nav-link.active {
+.page, .page-tabs .nav-tabs .nav-link.active, .btn-float-group {
```

Comment:
We want to avoid declaring any custom classes here, as this module should be restricted to overrides for Tabler itself. Maybe we can inherit the background color from `.page`?

Reply:
This has been moved to custom/_misc.scss. Inheriting from `.page` proved to be problematic as the class inherited many undesirable attributes.